tpm2tools/keys.go

```diff
@@ -273,7 +283,18 @@ func (k *Key) Unseal(in *tpmpb.SealedBytes, cOpt CertifyOpt) ([]byte, error) {
 		creationHash := sessionHashAlg.New()
 		creationHash.Write(in.GetCreationData())
 		if _, _, err = tpm2.CertifyCreation(k.rw, "", sealed, tpm2.HandleNull, nil, creationHash.Sum(nil), tpm2.SigScheme{}, ticket); err != nil {
-			return nil, fmt.Errorf("failed to certify creation: %v", err)
+			// Fail to CertifyCreation using a null key handle. It is possible that some firmware cannot take care of the
+			// null key handle.
+			// Try to CertifyCreation again with a dummy AIK key.
+			// Using the AIK is just a workaround, we still ignore the signature and attstation returned by CertifyCreation
```

Comment:
Let's rephrase this, something like
```
                There is a bug in some older vTPMs, where they are unable to
                CertifyCreation when using a Null signing handle (despite this
                being allowed by all versions of the TPM spec). To work around
                this bug, we use a temporary signing key and ignore the signed
                result. To reduce the cost of this workaround, we use a cached
                ECC signing key.
```

We should also wrap this comment so that it stays within 80-char lines (when using 8 space tabs)

Reply:
done